### PR TITLE
remove unused functions in volume/util

### PR DESCRIPTION
**Pull request by carlory**
#### What type of PR is this?

/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/volume/util/util.go [hunk 1]
```diff
@@ -59,10 +55,6 @@ const (
 	// managed by the attach/detach controller
 	ControllerManagedAttachAnnotation string = "volumes.kubernetes.io/controller-managed-attach-detach"
 
-	// MountsInGlobalPDPath is name of the directory appended to a volume plugin
-	// name to create the place for volume mounts in the global PD path.
-	MountsInGlobalPDPath = "mounts"
```
[the remark this answers is not in the document]
Comment (jsafrane) [reply pkg/volume/util/util.go:64]: The rest of the PR LGTM, thanks for the cleanup.

[the remark this answers is not in the document]
Comment (carlory) [reply pkg/volume/util/util.go]: @jsafrane updated.

**Pull request merged**